docs/crio.conf.5.md

```diff
@@ -141,6 +141,17 @@ the container runtime configuration.
 **blockio_config_file**=""
   Path to the blockio class configuration file for configuring the cgroup blockio controller.
 
+**cdi_spec_dirs**=[]
+  Directories to scan for Container Device Interface Specifications to enable CDI device injection. For more details about CDI and the syntax of CDI Spec files please refer to https://github.com/container-orchestrated-devices/container-device-interface.
+
+  Directories later in the list have precedence over earlier ones. The default directory list is:
+```
+  cdi_spec_dirs = [
+	  "/etc/cdi",
+	  "/var/run/cdi",
+  ]
+```
+
```

Comment:
Is there an associated man page that we could add too?

Reply:
Unfortunately currently there is no man page generated for/from the CDI Spec file documentation. I filed a related issue to CDI about it.